WebApiCustomers/Repository/BaseRepository.cs

```diff
@@ -0,0 +1,98 @@
+﻿using FluentValidation.Results;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WebApiCustomers.Model;
+using WebApiCustomers.Validator;
+
+namespace WebApiCustomers.Repository
+{
+    public class BaseRepository : IBaseRepository
+    {
+        private readonly List<CustomersModel> _customersList = new();
+
+        public virtual List<CustomersModel> GetAll()
+        {
+            return _customersList;
+        }
+
+        public virtual CustomersModel GetById(long id)
+        {
+            return _customersList.Where(x => x.Id == id).FirstOrDefault();
+        }
+
+        public virtual int Create(CustomersModel model)
+        {
+            model.Cpf = model.Cpf.Trim().Replace(".", "").Replace("-", "");
+            model.Id = _customersList.Count + 1;
+
+            CustomerValidator validator = new();
+            ValidationResult result = validator.Validate(model);
+
+            if (result.IsValid)
+            {
+                if (!_customersList.Any())
+                {
+                    _customersList.Add(model);
+                    return 201;
+                }
+
+                foreach (CustomersModel infoCustomer in _customersList)
+                {
+                    if (model.Cpf != infoCustomer.Cpf && model.Email != infoCustomer.Email)
+                    {
+                        _customersList.Add(model);
+                        return 201;
+                    }
+                    else
+                    {
+                        return 409;
+                    }
+                }
+            }
+            return 400;
```

Comment:
Ao invés de realizar essa verificação manualmente, é preferível realizar a verificação com o método Any.
```suggestion
                if(_customersList.Any(customer => customer.Cpf == model.Cpf || customer.Email == model.Email) return 409
                _customerList.Add(model);
                return 201;
```